katas/content/multi_qubit_systems/prepare_with_real/solution.md

```diff
@@ -1,12 +1,12 @@
-﻿Again, to start we will represent the goal state as a tensor product of single-qubit states; this gives us the following representation:
+﻿Again, to start you will represent the goal state as a tensor product of single-qubit states; this gives you the following representation:
 
 $$ \frac{1}{2}\big(\ket{00} - \ket{01} + \ket{10} - \ket{11}\big) = \frac{1}{2}\begin{bmatrix} 1 \\ -1 \\ 1 \\ -1 \end{bmatrix} = \frac{1}{\sqrt2} \begin{bmatrix} 1 \\ 1 \end{bmatrix} \otimes \frac{1}{\sqrt2}\begin{bmatrix} 1 \\ -1 \end{bmatrix} = \frac{1}{\sqrt2}\big(\ket{0} + \ket{1}\big) \otimes \frac{1}{\sqrt2}\big(\ket{0} - \ket{1}\big)  $$
 
-This time we need to transform both the first and the second qubits. Let's start with the first qubit. Applying the **H** gate transforms its state as follows:
+This time you need to transform both the first and the second qubits. Let's start with the first qubit. Applying the **H** gate transforms its state as follows:
```

Comment:
Should all references to quantum gates use the math context like this? I've seen that corrected elsewhere in this PR.
```suggestion
This time you need to transform both the first and the second qubits. Let's start with the first qubit. Applying the $H$ gate transforms its state as follows:
```